fs/expose/dokan/__init__.py

```diff
@@ -1034,29 +1067,29 @@ class Win32SafetyFS(WrapFS):
 
     """
 
-    def __init__(self,wrapped_fs,allow_autorun=False):
+    def __init__(self, wrapped_fs, allow_autorun=False):
         self.allow_autorun = allow_autorun
-        super(Win32SafetyFS,self).__init__(wrapped_fs)
+        super(Win32SafetyFS, self).__init__(wrapped_fs)
 
-    def _encode(self,path):
-        path = relpath(normpath(path))
-        path = path.replace(":","__colon__")
+    def _encode(self, path):
+        path = relpath(normpath(path.replace('\\', '/')))
+        path = path.replace(":", "__colon__")
         if not self.allow_autorun:
             if path.lower().startswith("_autorun."):
                 path = path[1:]
         return path
 
-    def _decode(self,path):
-        path = relpath(normpath(path))
-        path = path.replace("__colon__",":")
+    def _decode(self, path):
+        path = relpath(normpath(path.replace('\\', '/')))
```

Comment:
...and even if my previous comment is wrong, something here doesn't "look right" because `_encode` and `_decode` should be the inverse of each other.